openqa-trigger-bisect-jobs

```diff
@@ -110,9 +110,14 @@ client_args = [
 
 def call(cmds, dry_run=False):
     log.debug("call: %s" % cmds)
-    return subprocess.check_output(
-        (["echo", "Simulating: "] if dry_run else []) + cmds
-    ).decode("utf-8")
+    try:
+        return subprocess.check_output(
+            (["echo", "Simulating: "] if dry_run else []) + cmds,
+            stderr=subprocess.STDOUT, shell=True
+        ).decode("utf-8")
+    except subprocess.SubprocessError as err:
+        p_stderr = err.stderr if err.stderr else "No error putput"
+        raise RuntimeError(f"Command failed: {cmds[0]} ({p_stderr})") from err
```

Comment:
As already explained today please remove this. Only catch exceptions within `main`

Reply:
I made the change u asked. But I have to say I not sure about that. in theory you can call the openqa_clone or call in other places and then it wont catch the exception. I can think one or two other reasons but I think is minor